server/data/testData/incidentReporting.ts

```diff
@@ -115,6 +118,24 @@ export function mockReport({
   if (withDetails) {
     return {
       ...basicReport,
+      descriptionAddendums: withAddendums
+        ? [
+            {
+              createdBy: 'user1',
+              createdAt: format.isoDateTime(new Date('2023-12-05T15:34:00')),
```

Comment:
i think it's a better idea to base these dates on `incidentDateAndTime` so that the addenda aren't in the past